autoload/neomake.vim

```diff
@@ -62,6 +55,17 @@ function! neomake#has_async_support() abort
     return s:async
 endfunction
 
+if has('patch-7.3.1058')
+    function! s:function(name) abort
+        return function(a:name)
+    endfunction
+else
+    " Older Vim does not handle s: function references across files.
+    function! s:function(name) abort
+      return function(substitute(a:name,'^s:',matchstr(expand('<sfile>'), '.*\zs<SNR>\d\+_'),''))
+    endfunction
+endif
```

Comment:
The functions are not marked as covered - maybe a covimerage issue, because `s:function` is also used in utils..

Reply:
https://github.com/vim/vim/issues/3286 / https://github.com/vim-jp/issues/issues/1178